libkbfs/keybase_service_base.go

```diff
@@ -292,7 +292,14 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		Reason:        keybase1.IdentifyReason{Reason: reason},
 	}
 	res, err := k.identifyClient.Identify2(ctx, arg)
-	if err != nil {
+	// Identify2 still returns the user-plus-keys data, even if it
```

Comment:
@akalin-keybase: think of it like sharing before signup, except that the user has a Keybase username/UID.  The user has no keys, but we can still make an (empty) entry for their UID in the TLF metadata.